1/luke1.py

```diff
@@ -1,59 +1,62 @@
-import math
-
 coordinates = [(0,0)]
 
 currentx = 0
 currenty = 0
 
 facing = 0 # N, E, S, W => 0, 1, 2, 3
+found_twice = False
 
 
 def find_path(dir):
-    global facing, currentx, currenty
+    global facing, currentx, currenty, found_twice
     steps = int(dir[1:])
     if dir[0] == "R":
         facing = ((facing + 1 + 4) % 4)
     else:
         facing = ((facing - 1 + 4) % 4)
-    
+
     if facing == 0:
-        for i in xrange(steps):
+        for i in xrange(1, steps + 1):
             coordinate = (currentx, currenty + i)
             if coordinate not in coordinates:
                 coordinates.append(coordinate)
-            else:
-                print "Twice:", math.fabs(currentx)+math.fabs(currenty)
+            elif found_twice == False:
```

Comment:
`elif found_twice == False:` skrives vel enklere som `elif not found_twice:` :smile: